spacewalk/config/etc/httpd/conf.d/public.conf

```diff
@@ -0,0 +1,10 @@
+Alias /public /srv/www/htdocs/pub
+<Directory "/srv/www/htdocs/pub">
+    SetEnv VIRTUALENV
+    Options Indexes
+    Require all granted
+    AllowOverride All
+</Directory>
+
+RewriteEngine On
+RewriteRule ^/pub/(.+)$ /public/$1  [L,PT]
```

Comment:
With this line will be redirecting everything from `/pub/` to `/public`. 
What about the generated bootstrap scripts in this location, and which the customers can be relying one in their one environment?
https://documentation.suse.com/suma/4.3/en/suse-manager/reference/admin/bootstrap-script.html